src/main/java/frc/robot/targeting/VisionOutputs.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.targeting;
+
+import edu.wpi.first.math.geometry.Pose2d;
+
+public class VisionOutputs {
+    private Pose2d tagPosesRelativeToCamera;
+    private boolean cameraHasTargets;
+    private int tagIDs;
+    private double latency;
+
+    public double getLatency() {
+        return this.latency;
+    }
+
+    public void setLatency(double latency) {
+        this.latency = latency;
+    }
+
+    public int getTagIDs() {
+        return this.tagIDs;
+    }
+
+    public void setTagIDs(int tagIDs) {
+        this.tagIDs = tagIDs;
+    }
+
+    public boolean isCameraHasTargets() {
```

Comment:
prefer cameraHasTargets.  even better, forget about camera-- the user of this object doesnt care, and simply use hasTargets().  see above, better to answer hasTargets() by the length of the list of targets you ahve